src/test/java/org/odpi/openmetadata/adapters/connectors/integration/lineage/sample/sampleLineageEventProcessorTest.java

```diff
@@ -27,71 +27,70 @@
 
 
 /**
- *  Test of the event parsing into the EventContent object
+ * Test of the event parsing into the EventContent object
  */
-public class sampleLineageEventProcessorTest
-{
+public class SampleLineageEventProcessorTest {
```

Comment:
this rename of the class - causes it to mismatch the file name so it does not build